components/todo-input.js

```diff
@@ -1,20 +1,110 @@
-import React from "react";
+import React, { useState } from 'react';
+
 import styled from "styled-components";
 
-export default function TodoInput() {
+export default function TodoInput({addTodo}){
+
+  const [newTodo, setNewTodo] = useState({});
+
+  const changeInputData = (e) => {
+    setNewTodo({...newTodo, [e.target.name]:e.target.value});
+  }
+
   return (
-    <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
-    </Wrapper>
+    <InputWrapper>
+      <TimeBox>
+        <Time>시간</Time>
+        <InputDay name="date" type="number" placeholder="날짜를 입력하세요 (ex.20200404)" 
+        onInput = {(e) =>{e.target.value = Math.max(0, parseInt(e.target.value) ).toString().slice(0,8)}}
+        onChange={changeInputData}
+        />
+      </TimeBox>
+      <TodoBox>
+        <Todo>TODO</Todo>
+        <InputTodo name="content" placeholder='할 일을 입력하세요' onChange={changeInputData}></InputTodo>
+      </TodoBox>
+      <EnterBtn onClick={() => addTodo({...newTodo})}>등록</EnterBtn>
+    </InputWrapper>
   );
 }
 
-const Wrapper = styled.div`
-  border: solid 1px;
-  font-size: 18px;
-  flex: 1;
+
+const InputWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  -webkit-box-align: center;
+  align-items: center;
+  height: 37rem;
+  flex: 1 1 0%;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+  border-image: initial;
+  padding: 1rem 2rem;
+`;
```

Comment:
```suggestion
const Wrapper = styled.div`
  width: 100%;
  display: flex;
  flex-direction: column;
  align-items: center;
  height: 37rem;
  flex: 1;
  border: 1px solid black;
  padding: 1rem 2rem;
`;
```
1. 가장 바깥 Wrapper는 굳이 다른 단어 붙이실 필요 없이 ```Wrapper```라고 써주시면 됩니다.
어떤 역할을 하는지는 파일 이름에서 알 수 있기 때문이죠!
2. box-align은 표준 문법이 아니기 때문에 사용 안 하시는 게 좋습니다. [참고](https://developer.mozilla.org/en-US/docs/Web/CSS/box-align)
3. ```flex : 1 1 0%```는 ```flex : 1```과 같습니다.  [참고](https://heropy.blog/2018/11/24/css-flexible-box/#flex)
4. ```border-width: 1px, border-style: solid, border-color: black```를 ```border: 1px solid black```으로 가독성 좋게 쓸 수 있습니다.
5. border-image는 화면 스타일을 구현하는 데에 필요 없는 코드입니다.